NetKAN/REKT.netkan

```diff
@@ -0,0 +1,19 @@
+{
+  "spec_version": "v1.4",
+  "depends": [
+    { "name": "ModuleManager" },
+    { "name": "RetractableLiftingSurface" }
+  ],
+  "install": [
+    {
+      "install_to": "GameData",
+      "find": "REKT-0.4.3.1",
+      "filter": ["RetractableLiftingSurface.version"]
+    }
```

Comment:
This is going to install two copies of RetractableLiftingSurface, one in the REKT folder and one outside it (from the dependency).
Try this:

    "install": [
        {
            "install_to": "GameData",
            "find_regexp": "REKT-",
            "filter": "RetractableLiftingSurface"
        }
    ],